azure/durable_functions/models/DurableOrchestrationContext.py

```diff
@@ -471,6 +491,37 @@ def parent_instance_id(self) -> str:
         """
         return self._parent_instance_id
 
+    @property
+    def maximum_short_timer_duration(self) -> datetime.timedelta:
```

Comment:
With this change, are we effectively adding new public APIs? I hope we wouldn't make them public, and that we'd instead just use the underlying values internally. If these properties need to be here, could we prefix them with `_` to make it clear that they are internal?

Reply:
I do think prefixing with `_` makes the most sense here - though it is worth noting that there are several other @property fields on this object which I don't think are used by customers - context.histories and context.function_context, to name a few. Maybe there are uses for these that I am not aware of

Reply:
Edit - class members declared in this way, with `@property` on a method, cannot begin with `_`. I have removed the accessor definitions alltogether, as these should not be exposed as part of the API anyway, and all references now use the `_` prefixed property defined in `__init__`

Reply:
OK, so it sounds like this isn't *actually* a public API, and we declare other non-public properties this way, so nothing to worry about.